docs/internals/security.txt

```diff
@@ -49,8 +49,13 @@ requires a security release:
 * The vulnerability is within a :ref:`supported version <security-support>` of
   Django.
 
-* The vulnerability applies to a production-grade Django application. This means
-  the following do not require a security release:
+* The vulnerability does not depend on manual actions performed by a project's
+  developer or maintainer through developer tools or the Django CLI that rely
+  on code external to Django. For example, attacks requiring the execution of
+  management commands with uncommon or insecure options do not qualify.
```

Comment:
I like the idea, but the phrasing seems cumbersome. I tried to rearrange it a little.

```suggestion
* The vulnerability does not depend on manual actions that rely on code
  external to Django. Manual actions, here, refers to actions performed by
  a project's developer or maintainer, through developer tools or the Django
  CLI.  For example, attacks requiring the execution of management 
  commands with uncommon or insecure options do not qualify.
```